client/extensions/woocommerce/components/product-search/results.js

```diff
@@ -0,0 +1,103 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import {
+	areProductSearchResultsLoaded,
+	areProductSearchResultsLoading,
+} from 'woocommerce/state/sites/products/selectors';
+import { getProductSearchResults } from 'woocommerce/state/ui/products/selectors';
+import ProductItem from './item';
+
+class ProductSearchResults extends Component {
+	static propTypes = {
+		onSelect: PropTypes.func.isRequired,
+		search: PropTypes.string.isRequired,
+	};
+
+	renderLoading = () => {
+		return (
+			<div className="product-search__results is-placeholder">
+				<CompactCard className="product-search__item">
+					<div className="product-search__image">
+						<span />
+					</div>
+					<div className="product-search__label">
+						<div className="product-search__name">
+							<span />
+						</div>
+						<div className="product-search__sku">
+							<span />
+						</div>
+					</div>
+				</CompactCard>
+			</div>
+		);
+	};
+
+	renderNotFound = () => {
+		const { translate } = this.props;
+		return (
+			<CompactCard className="product-search__item">
+				<div className="product-search__image">
+					<Gridicon icon="info-outline" />
+				</div>
+				<div className="product-search__label">
+					<div className="product-search__name">{ translate( 'No products found' ) }</div>
```

Comment:
Nit: If this happens, probably don't want cursor pointer?

![img_0258](https://user-images.githubusercontent.com/1595739/30984860-77e4b2c0-a443-11e7-8c87-5e820cf81025.JPG)